lib/tasks/plugin_assets.rake

```diff
@@ -15,6 +15,11 @@ task 'plugin:assets:precompile', [:plugin] => [:environment] do |t, args|
         app = Rails.application
         app.config.assets.digest = true
         app.config.assets.precompile = plugin.assets
+        if plugin.gettext_domain
+          prefix = plugin.engine.root.join('app/assets/javascripts/')
+          locale_assets = Dir[prefix.join("locale/**/#{plugin.gettext_domain}.js")].map { |p| p[prefix.to_s.length..-1] }
```

Comment:
I think this ends up with with `/locale/..` and https://guides.rubyonrails.org/asset_pipeline.html#precompiling-assets only has relative paths.

It also has this bit:

> The default matcher for compiling files includes application.js, application.css and all non-JS/CSS files (this will include all image assets automatically) from app/assets folders including your gems:
```
[ Proc.new { |filename, path| path =~ /app\/assets/ && !%w(.js .css).include?(File.extname(filename)) },
/application.(css|js)$/ ]
```

That does answer my earlier question on why it's needed.

Thinking more about this, I think we should do it here:

https://github.com/theforeman/foreman/blob/2dc52832034fb8dc289819ebc04f26af13902d33/config/initializers/assets.rb#L22-L23

Reply:
> I think this ends up with with `/locale/..`

```
> locale_assets.take(3)
=> ["locale/en/foreman_tasks.js", "locale/fr/foreman_tasks.js", "locale/ja/foreman_tasks.js"]
```

It doesn't seem to

> Thinking more about this, I think we should do it here:

That doesn't work. We could be setting it there, but then in plugin_assets L17 we drop whatever is in there and use only plugin's assets, which currently doesn't include these locale files

Reply:
That looks like the cleanest way out